code/__DEFINES/sort_types.dm

```diff
@@ -0,0 +1,33 @@
+
+#define SORT_TYPE_WASTE 0
```

Comment:
i wish these could be strings (i'm not fond of funny numbers) but that's probably out-of-scope for the sake of this PR and would require some deep work on disposals code

Reply:
I wanted to do it this PR, but i worried for downstreams...

Reply:
they knew what they were signing up for.

if you are deeply worried, you could probably spend a few hours cracking out an updatepaths with property filtration and multiple-path-output

https://github.com/tgstation/tgstation/tree/master/tools/UpdatePaths#multiple-path-output
https://github.com/tgstation/tgstation/tree/master/tools/UpdatePaths#method-open-mind-to-all-possibilities

So maybe like:
```txt
/obj/machinery/disposals_pipe{sorttype = 1} : /obj/machinery/disposals_pipe{@OLD, sorttype=@SKIP}, /obj/effect/mapping_helper/waste
```

I just made up those names in this scenario, but maybe try working with that? stringifying from there would be easier i think since it'll just be code-side stored in the helper and you don't have to overwrite all the maps again.

Reply:
That could work very well, but sorttype sadly, can be either a number of a semicolon delimited list. Still, thank you for pointing me to the UpdatePath's documentation, I have been meaning to find it, both for our downstream and also for potentially renaming a few of these helpers I have made. 👍

Reply:
Well that could also be just explicit matching for all the potential cases...

```txt
/obj/machinery/disposals_pipe{sorttype = 1;2;3} :  all output paths here
```

but that's probably way too much work given how many potential combinations you can have

Reply:
:pensive: I guess I will just do it the hard way in a future PR.